tests/dyn-drv/text-hashed-output.nix

```
@@ -0,0 +1,29 @@
+with import ./config.nix;
+
+# A simple content-addressed derivation.
+# The derivation can be arbitrarily modified by passing a different `seed`,
+# but the output will always be the same
+rec {
+  root = mkDerivation {
```

```suggestion
  hello = mkDerivation {
```

Graph connotations aren't helpful here. It's only a root of the reverse dependency graph.